examples/example-generation/examples/gp.sg

```diff
@@ -0,0 +1,56 @@
+# Importing generated data
+CSVFile X("../data/gaussian_process_X.csv")
+CSVFile X_test("../data/gaussian_process_X_test.csv")
+CSVFile Y("../data/gaussian_process_Y.csv")
+CSVFile Y_test("../data/gaussian_process_Y_test.csv")
+
+RegressionLabels labels(Y)
+RegressionLabels labels_test(Y_test)
+RealFeatures feats_train(X)
+RealFeatures feats_test(X_test)
+
+# GP specification
+GaussianKernel kernel(10, 2)
```

Comment:
I think it is ok to drop them or make the keyword a comment. Don't want to push this but if it is easy - would be cool